public/angularjs/file.js

```diff
@@ -16,119 +16,119 @@ app.directive('fileChange', ['$window', ($window) => {
         } else if (!fileContents) {
           swal({
```

Comment:
<code>swal</code> function comes from the sweet alert library that I have imported in the HTML file as CDN.Sweet alert is a replacement of the JS alerts.It produces nice bootstrapped modals that are easily customizable according to the need and message one needs, for example <code>error,warning</code> or just <code>info</code>. check this[ link](http://t4t5.github.io/sweetalert/) for more info.